lib/galaxy/webapps/galaxy/api/histories.py

```diff
@@ -246,6 +246,7 @@ def published( self, trans, **kwd ):
             rval.append( history_dict )
         return rval
 
+    # TODO: does this need to be anonymous_and_sessionless? Not just expose_api?
```

Comment:
For reference, this was added in https://github.com/galaxyproject/galaxy/pull/614.